crates/shared/src/maintenance.rs

```diff
@@ -34,19 +63,33 @@ impl Maintaining for ServiceMaintenance {
             if let Err(err) = result {
                 tracing::warn!("Service Maintenance Error: {:?}", err);
```

Comment:
Could you also add the name of service in this error message?

Reply:
Ah yes, forgot that. Thanks

Reply:
Reopening since fix is not correct.

Reply:
Now it's fixed properly.